_docs/_latest/setup/upgrade.md

```diff
@@ -151,6 +158,271 @@ change a certain resource, you'll need to create a new deployment of Forseti.
 
 Learn more about [Updating a Deployment](https://cloud.google.com/deployment-manager/docs/deployments/updating-deployments).
 
+{% capture upgrading_2_0_0_to_2_1_0 %}
+
+1. Open cloud shell when you are in the Forseti project on GCP.
+1. Checkout forseti with tag v2.1.0 by running the following commands:
+    1. If you already have the forseti-security folder under your cloud shell directory, 
+    run command `rm -rf forseti-security` to delete the folder.
+    1. Run command `git clone https://github.com/GoogleCloudPlatform/forseti-security.git` to 
+    clone the forseti-security directory to cloud shell.
+    1. Run command `cd forseti-security` to navigate to the forseti-security directory.
+    1. Run command `git checkout tags/v2.1.0` to checkout version `v2.1.0` of Forseti Security.
+1. Download the latest copy of your Forseti server deployment template file from the Forseti server GCS 
+bucket (located under `forseti-server-xxxxxx/deployment_templates`).
+1. Update the `forseti-version` inside the deployment template to `tags/v2.1.0`.
+1. Create file `deploy-forseti-server-xxxxx-2-1-0.yaml` under `forseti-security/deployment-templates`.
+1. Copy and paste the content of the deployment template to file `deploy-forseti-server-xxxxx-2-1-0.yaml`.
```

Comment:
Per offline discussion, it would be better to just use gsutil command which can create the new file with the correct name in one shot in cloud shell env, and save the copy and paste step from local env to cloud shell.